x-pack/plugins/alerting/server/lib/is_alert_not_found_error.ts

```diff
@@ -0,0 +1,11 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { SavedObjectsErrorHelpers } from '../../../../../src/core/server';
+
+export function isAlertSavedObjectNotFoundError(err: Error, alertId: string) {
+  return SavedObjectsErrorHelpers.isNotFoundError(err) && `${err}`.includes(alertId);
```

Comment:
This feels yucky enough that I feel like we should ask for a better helper - like being able to pass `alertId` into that `isNotFoundError()` function, rather than text searching the error message ourselves.  The `alertId` values are UUIDs today, which seems safe enough to test for, but who knows what tomorrow holds - action id's can now be user-specified via pre-configured actions ...

Reply:
Yeah, that's fair, but as there seems to be pressure to get this backported to 7.7, I think we'll have to defer that.